spec/factories/room.rb

```diff
@@ -16,7 +16,7 @@
     end
 
     trait :with_description do
-      description { Faker::TvShows::TheExpanse.quote }
+      description { Faker::TvShows::TheExpanse.quote.truncate(300) }
```

Comment:
Could this value be stored in a constant, like `Room::DescriptionLength`? It feels magical here.

Reply:
I could go with `Room::MAXIMUM_DESCRIPTION_LENGTH`.

Reply:
I went with `DESCRIPTION_MAX_LENGTH` so the starting letter for the constant matches the attribute it is used to validate.